spec/unit/shoulda/matchers/active_model/validate_confirmation_of_matcher_spec.rb

```diff
@@ -13,10 +13,13 @@
   end
 
   context 'when the model has a confirmation validation' do
-    it 'passes' do
-      builder = builder_for_record_validating_confirmation
-      expect(builder.record).
-        to validate_confirmation_of(builder.attribute_to_confirm)
+    [:string, :integer].each do |column_data_type|
+      it "passes when data type is #{column_data_type}" do
```

Comment:
Similar thing here, let's rename `column_data_type to simply `column_type`.

And then perhaps: "it passes when the column is of type #{column_type}"